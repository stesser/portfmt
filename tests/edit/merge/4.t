printf "CARGO_CRATES=" | ${PORTEDIT} merge $input | diff -L $expected -L $actual -u $expected -
<<<<<<<<<
PORTNAME=

# enable all output serialization formats
CARGO_FEATURES=	all
<<<<<<<<<
PORTNAME=

CARGO_CRATES=

# enable all output serialization formats
CARGO_FEATURES=	all
