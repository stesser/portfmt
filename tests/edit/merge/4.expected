PORTNAME=

CARGO_CRATES=

# enable all output serialization formats
CARGO_FEATURES=	all
